channels/xrdpvr/xrdpvr_player.c

```diff
@@ -141,7 +150,7 @@ typedef struct player_state_info
     AVCodec         *video_codec;
     AVFrame         *audio_frame;
     AVFrame         *video_frame;
-#if defined(DISTRO_DEBIAN8)
+#if defined(DISTRO_DEBIAN8) || defined(DISTRO_UBUNTU1604)
```

Comment:
I know its already there but I don't like those `DISTRO_xxx` backport style. its very specific.
we should backport the code so it builds everywhere with this ffmpeg libs 

there's not really an dependency/change on a specific distro

Reply:
you can keep it like this for now. meh.